marklogic-data-hub-central/build.gradle

```diff
@@ -194,6 +194,13 @@ task copyUiFiles(type: Copy, dependsOn: deleteSpringBootUiFiles, group: taskGrou
     into springBootUiPath
 }
 
+task lintUI(type: NpmTask, dependsOn: installDependencies, group: taskGroup) {
+    description = "linting the UI files"
```

Comment:
Can you expand the description to define what happens when the task is run? For example, if no errors are found, is any report file written? If errors are found, what happens? Does the task fail, or does it print a report somewhere?

Reply:
Before committing description here is the current implementation: 
    - if no errors, the success email is sent with message 'TestUnit passed'. 
    - If lint task is failed the job is terminated as failure and no notifications are sent.